### ISSUE TITLE
2018 Sprint Goals & Metrics

### ISSUE THREAD

**Issue by zee-moz** [mozilla/2018-global-sprint-planning#2]
Note: these are not necessarily public facing goals and metrics-- these are internal benchmarks for 2018. We will collect this data, but aren't limited to it.We will measure all activity by all participants (both contributors and project leads) across all projects in the Sprint, but specifically we look for: 

**Project and participation #s**
 75% increase: 122 sites; 189 projects. 

** Project Activity Metric: increased and sustained activity on all projects**
Compare average activity across all projects:
- 1 week before the sprint
- during the sprint,
- 6 weeks after (for sustained contribution
- **Goal: activity up 300% during sprint, up 25% after**

**Contributor Growth Metric: increase in # of high-engagement events by contributors who are during the sprint**
_(note here "contributors" means those coming to a project, vs those leading that project)_
- Issues opened or closed
- Pull requests opened, closed, or reopened
- **Goal: increase # of meaningful contribution events by 50% over last year**

**Comment by zee-moz** [comment]
@acabunoc can you summarize the process and specs for this data analysis project so I can work on getting dev time from Alan?

**Comment by zee-moz** [comment]
@alanmoo here are the metrics and the plan

**Comment by abbycabs** [comment]
data collected from last year: https://github.com/mozilla/global-sprint/tree/2017/_data/activity

**Comment by abbycabs** [comment]
https://medium.com/read-write-participate/mozsprint-by-the-github-numbers-49d518551725

**Comment by abbycabs** [comment]
Endpoint used: https://developer.github.com/v3/activity/events/

**Comment by Saallen** [comment]
**First draft -Goals for event**

_External_ --To faciliate connections, collaborations and the growth of open source projects and contributions. 
_Internal_ -- Opportunity for leadership development for many project leads who take their project from ideation stage to public contribution, or from local community project to global contributions. Also great potential for experienced mentors to support the delivery and the growth of a small group of project and their leads as the develope along their open sourse and leadership path

**Comment by zee-moz** [comment]
@Saallen thanks, these are great! Can you also respond to the Project/Participation numbers and the Contributor Growth Metric listed in the issue? Feedback on these is pressing as we're designing data collection around them. Thanks! 😺

**Comment by Saallen** [comment]
Hello
** Project Activity Metric: increased and sustained activity on all projects**
    1 week before the sprint- why do we need to measure here? as surely everyone will just be prepping their projects??Can you share the base line here from 2017?

Can you expand what you mean in the second goal by events?

**Comment by zee-moz** [comment]
@Saallen  events are things people do on GitHub to meaningfully contribute:

- 1. Issues opened or closed
- 2. Pull requests opened, closed, or reopened

As for the 1 week timeline-- some projects in 2018 may be new and not have 2017 activity/history.
If you are talking about 2017 baseline activity on 2017 projects from Sprint (?) I think that data can't be retroactively collected, you have to collect it at the time... @acabunoc will know more. 

This 1-week collection period should be past our featured project deadlines, so most project prep should be done. We can also separate out activity by leads vs. contributors, to screen out activity that's just prep if we're concerned about that noise (though I know that Steph would argue that any Sprint-related project activity is a plus). @acabunoc what do you think?
